app/logic/fileHandler.py

```
@@ -15,11 +15,16 @@ def __init__(self,files=None, courseId=None, eventId=None):
             # eventID is not included in the path, because it is now a part of the attachment filename.
             self.path = os.path.join(self.path, app.config['files']['event_attachment_path'])
         
+    def makeDirectory(self):
+        # Creating the directory when call in saveFiles
         try:
             extraDir = str(self.eventId) if self.eventId else ""
             os.makedirs(os.path.join(self.path, extraDir))
-        except:
-            print("Directory exists.")
+        # Occurs when we try to create a directory that already exists
+        except OSError as e:
+            if e.errno != 17:
+                pass
```

This try/except is not doing what you want. You wanted it to ignore only error 17, but what you are doing is ignoring all `OSError` exceptions, and the conditional doesn't matter. You need to re-raise the exception if you don't want to catch it, not pass it.